specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.ApiManagement/client.tsp

```diff
@@ -155,7 +155,16 @@ using Azure.ClientGenerator.Core;
 @@usage(Microsoft.EventGrid.SystemEvents.ApiManagementGatewayApiRemovedEventData,
   Usage.output
 );
-@@access(Microsoft.EventGrid.SystemEvents.ApiManagementGatewayApiRemovedEventData,
+@@access(Microsoft.EventGrid.SystemEvents.ApiManagementCircuitBreakerOpenedEventData,
+  Access.public
```

Comment:
It looks like we are missing @@usage here for these 4 events: i.e.

@@usage(Microsoft.EventGrid.SystemEvents.ApiManagementCircuitBreakerOpenedEventData,
  Usage.output);

Can we add these?